dashboard/src/shared/I18n.ts

```diff
@@ -0,0 +1,45 @@
+import messages_en from "../locales/en.json";
+import { axiosWithAuth } from "./AxiosInstance";
+
+export interface II18nConfig {
+  locale: string;
+  messages: Record<string, string>;
+}
+
+export enum ISupportedLangs {
+  en = "en",
+  custom = "custom",
```

Comment:
isn't it better to leave it just with `en`?

Reply:
We use it when loading the custom one (from values.yaml); don't know why we should remove it

Reply:
I thought this worked differently. I'd assume that you have the default language there (`en`) and you could overwrite it with the one from the chart. Isn't that the case?

Reply:
Ahh, no no. The current impl is:
(side note: locales != messages; the former is about date, number, time, etc formats)

1. Are the custom messages used? 
  - Yes: use custom: locale=english, messages=custom
    - (runtime): have I the custom message for the string X?
      - Yes: use custom message
      - No: use default harcoded message
  - No:  GOTO 2
2. What is the current language user configuration? Is the ISO code one of the ISupported?
  - Yes: use this lang: locale=lang, messages=lang
  - No: use default: locale=english, messages=english

(This behavior should be also documented, I guess :P)

Reply:
isn't it easier to just have `en` rather than `en` + `custom`?

Reply:
I don't get your point here? Why would it be easier? 
I mean, we can remove it and pass the messages directly in [this line](https://github.com/kubeapps/kubeapps/pull/2347/files#diff-f822cbb1da0404c050e7403bee68b227a580cfce308444f6330b57acd38de6c7R34), but what's the benefit?

Reply:
yep, getting rid of `getCustomI18nConfig` and supporting just one language (but I may be missing something)

Reply:
In any case, I am not feeling strong about this so what go with what you prefer.